airflow/sensors/sql_sensor.py

```diff
@@ -102,6 +98,4 @@ def poke(self, context):
                 return self.success(first_cell)
             else:
                 raise AirflowException("self.success is present, but not callable -> {}".format(self.success))
-        if self.allow_null:
-            return str(first_cell) not in ('0', '')
         return str(first_cell) not in ('0', '', 'None')
```

Comment:
Waaaait. I just noticed. This means if I do `INSERT VALUES("None")` in my table it would fail. That is a bug in the original PR that we need to fix first.

Reply:
(By first I mean as a separate PR that we can apply to the 1.10.x release)

Reply:
I can't figure out how to fix already merged pull request. (btw fixed in current PR)

Reply:
There's no way to change the merged PR. You'd have to create a new PR with that change please.

Reply:
I'm confused a little bit. That PR should be from `v1-10-stable` branch, right?

Reply:
No, just a separate PR agsint the master branch (as we need the change against  this line - `str(first_cell) not in ('0', '', 'None')`) Once that is in master I will cherry-pick it back to the release branch.

Reply:
fixed here https://github.com/apache/airflow/pull/6336